src/common/actions/repositories.js

```diff
@@ -42,44 +39,27 @@ export function fetchUserRepositories(pageNumber) {
           if (result.status !== 'success') {
             throw getError(response, result);
           }
-          dispatch(setRepositories(result.payload.repos));
-          dispatch(fetchRepositoriesSuccess());
```

Comment:
I don't think we need a separate reducer for success, it's implicit in getting back results in this case.